docker/legacy/Dockerfile.s3

```diff
@@ -0,0 +1,21 @@
+ARG BASE_IMAGE=confluentinc/cp-kafka-connect-base:7.8.1
+
+FROM --platform=$BUILDPLATFORM gradle:8.9-jdk17 AS builder
+
+COPY ./*.gradle /code/
+COPY src/main/java /code/src/main/java
+WORKDIR /code
+RUN gradle jar --no-watch-fs
+
+FROM ${BASE_IMAGE}
+
+ENV WAIT_FOR_KAFKA="1"
+ARG KAFKA_CONNECT_S3_VERSION="10.5.23"
+
+COPY --from=builder /code/build/libs/kafka-connect-transform-keyvalue*.jar /usr/share/"${COMPONENT}"/plugins/
+COPY ./docker/legacy/launch /etc/confluent/docker/launch
+COPY ./docker/legacy/kafka-wait /usr/bin/kafka-wait
```

Comment:
the relative paths here need to be updated, since this Dockerfile was also moved /docker/legacy

Reply:
You're right, it is correct as is. I just assumed that since this Dockerfile was moved to /docker/legacy/, the build context would be also from inside that folder, so that the relative paths to launch and kafka-wait would be different (as it is in the same folder). I just realized that the project root needs to be kept as build context root in order to reach the /src/ location.